roles/confluent.common/tasks/main.yml

```diff
@@ -13,17 +8,23 @@
   loop:
     - ansible_os_family
     - ansible_distribution
+    - ansible_distribution_major_version
     - ansible_system
     - ansible_architecture
 
-# JVM Installation and Repo Setup
-- include_tasks: redhat.yml
+- name: Host Validations
+  include_tasks: precheck.yml
```

Comment:
Maybe select another filename? The term "precheck" could represent anything. Maybe `host_validations.yml`?

Reply:
sure, I like host validations, except in the future we may also do variable validations in this task file. as in if are you setting to contradictory vars, error out. 

Oh checking ports are open is on the roadmap too so that'd be "host validation"

Any other file name ideas that contain both host validations and variable validations?

Reply:
Why do you want to put different things in the same file? IMO validating host requirements and variables are different things.

Reply:
So in general i follow the pattern:
```
include_tasks: tasks.yml
when: reason-to-skip
```
that all tasks that will get skipped by one condition should be grouped into one file to minimize files/lines of code.

Now you have a point that variable validations may not even happen in this role/ they arent written yet so we can cross that bridge later. I'll change to host_validations.yml